packages/runtime/container-runtime/src/gc/gcHelpers.ts

```diff
@@ -296,3 +297,13 @@ export function unpackChildNodesGCDetails(gcDetails: IGarbageCollectionDetailsBa
 export function trimLeadingAndTrailingSlashes(str: string) {
 	return str.replace(/^\/+|\/+$/g, "");
 }
+
+/**
+ * Wrap the given value in a CodeArtifact tag.
+ */
+export function wrapWithCodeArtifact(value: string) {
```

Comment:
How about `tagAsCodeArtifact`?

Reply:
Or `tryTagCodeArtifact` and pass-through `undefined`, since there are a few callsites that are having to check for `undefined`?

Reply:
Passing through undefined means the call sites have to check the return value, So, just renamed to `tagAsCodeArtifact`